ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/CancunTargetingGasLimitCalculator.java

```diff
@@ -15,17 +15,36 @@
 package org.hyperledger.besu.ethereum.mainnet;
 
 import org.hyperledger.besu.ethereum.mainnet.feemarket.BaseFeeMarket;
+import org.hyperledger.besu.evm.gascalculator.CancunGasCalculator;
 
 public class CancunTargetingGasLimitCalculator extends LondonTargetingGasLimitCalculator {
-  private static final long MAX_BLOB_GAS_PER_BLOCK = 786432L;
+
+  /**
+   * The constant MAX_BLOB_GAS_PER_BLOCK represents the maximum gas limit for blob data. =
```

Comment:
comment needs adjusting since this is now max blobs not max blob gas

Reply:
adjuste